File: itJob/src/main/java/com/itJob/bean/Enterprise.java
package com.itJob.bean;

import com.baomidou.mybatisplus.annotations.TableField;
import com.baomidou.mybatisplus.annotations.TableId;
import com.baomidou.mybatisplus.annotations.TableName;
import com.baomidou.mybatisplus.enums.IdType;
import lombok.Data;

/**
 * @Description: 企业 类
 * @Author: LRJ
 * @Date: 2019/12/27 18:03
 */
@Data
@TableName("t_enterprise")
public class Enterprise {
    @TableId(value = "enterprise_id",type = IdType.UUID)
    String id;
    String userId;
    @TableField(value ="enterprise_name")
    String name;
    String icon;
    String type;
    String financing;
    String phone;
    String email;
    String address;
    String introduction;

}

File: itJob/WeChat/models/enterprise.js
import { HTTP } from '../utils/http.js'
class Enterprise extends HTTP {
  getEnterpriseById(id, sCallback) {
    this.request({
      url: '/enterprise/getOneById',
      method: 'get',
      data: {
        id: id,
      },
      success: (res) => {
        sCallback(res)
      }
    })
  }
  getEnterpriseByUserId(id, sCallback) {
    this.request({
      url: '/enterprise/getOneByUserId',
      method: 'get',
      data: {
        userid: id,
      },
      success: (res) => {
        sCallback(res)
      }
    })
  }
  updateEnterprise(person, sCallback) {
    this.request({
      url: '/enterprise/updateEnterprise',
      method: 'post',
      data: {
        id: person.id,
        userId: person.userid,
        name: person.name,
        icon: person.icon,
        type: person.type,
        financing: person.financing,
        phone: person.phone,
        address: person.address,
        email: person.email,
        introduction: person.introduction,
      },
      success: (res) => {
        sCallback(res)
      }
    })
  }
  insertEnterprise(person, sCallback) {
    this.request({
      url: '/enterprise/insertEnterprise',
      method: 'post',
      data: {
        userid: person.userid,
        name: person.name,
        icon: person.icon,
        type: person.type,
        financing: person.financing,
        phone: person.phone,
        address: person.address,
        email: person.email,
        introduction: person.introduction,
      },
      success: (res) => {
        sCallback(res)
      }
    })
  }
}
export { Enterprise }